torch/csrc/autograd/function.h

```diff
@@ -523,6 +523,12 @@ struct TORCH_API Node : std::enable_shared_from_this<Node> {
     return tensor_pre_hooks_;
   }
 
+  virtual std::vector<std::unique_ptr<PostAccumulateGradHook>>&
+  tensor_post_acc_grad_hooks() noexcept {
```

Comment:
I'm not sure what this is for tbh
Could we remove this?

Reply:
Compiled autograd is looping through the nodes generically in the graph and getting compiled_args for each of their hooks 
So this is useful to be called there as well so that we can produce an explicit error for that case.